packages/docusaurus-theme-translations/locales/pl/theme-search-algolia.json

```
@@ -1,29 +1,29 @@
-  "theme.SearchModal.footer.closeKeyAriaLabel": "Escape key",
-  "theme.SearchModal.footer.closeText": "to close",
-  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Arrow down",
-  "theme.SearchModal.footer.navigateText": "to navigate",
-  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Arrow up",
-  "theme.SearchModal.footer.searchByText": "Search by",
-  "theme.SearchModal.footer.selectKeyAriaLabel": "Enter key",
-  "theme.SearchModal.footer.selectText": "to select",
-  "theme.SearchModal.noResultsScreen.noResultsText": "No results for",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Let us know.",
-  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Believe this query should return results?",
-  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Try searching for",
-  "theme.SearchModal.placeholder": "Search docs",
-  "theme.SearchModal.searchBox.cancelButtonText": "Cancel",
-  "theme.SearchModal.searchBox.resetButtonTitle": "Clear the query",
-  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Favorite",
-  "theme.SearchModal.startScreen.noRecentSearchesText": "No recent searches",
-  "theme.SearchModal.startScreen.recentSearchesTitle": "Recent",
-  "theme.SearchModal.startScreen.removeFavoriteSearchButtonTitle": "Remove this search from favorites",
-  "theme.SearchModal.startScreen.removeRecentSearchButtonTitle": "Remove this search from history",
-  "theme.SearchModal.startScreen.saveRecentSearchButtonTitle": "Save this search",
+  "theme.SearchModal.errorScreen.helpText": "Może warto sprawdzić połączenie sieciowe.",
+  "theme.SearchModal.errorScreen.titleText": "Nie można pobrać wyników",
+  "theme.SearchModal.footer.closeKeyAriaLabel": "Klawisz Escape",
+  "theme.SearchModal.footer.closeText": "do zamknięcia",
+  "theme.SearchModal.footer.navigateDownKeyAriaLabel": "Strzałka w dół",
+  "theme.SearchModal.footer.navigateText": "poruszać się",
+  "theme.SearchModal.footer.navigateUpKeyAriaLabel": "Strzałka w górę",
+  "theme.SearchModal.footer.searchByText": "Szukaj według",
+  "theme.SearchModal.footer.selectKeyAriaLabel": "Klawisz Enter",
+  "theme.SearchModal.footer.selectText": "aby wybrać",
+  "theme.SearchModal.noResultsScreen.noResultsText": "Brak wyników dla",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsLinkText": "Daj nam znać.",
+  "theme.SearchModal.noResultsScreen.reportMissingResultsText": "Uwierz, że to zapytanie powinno zwrócić wyniki?",
+  "theme.SearchModal.noResultsScreen.suggestedQueryText": "Spróbuj poszukać",
+  "theme.SearchModal.placeholder": "Wyszukaj dokumenty",
+  "theme.SearchModal.searchBox.cancelButtonText": "Anuluj",
+  "theme.SearchModal.searchBox.resetButtonTitle": "Wyczyść zapytanie",
+  "theme.SearchModal.startScreen.favoriteSearchesTitle": "Ulubione",
+  "theme.SearchModal.startScreen.noRecentSearchesText": "Brak ostatnich wyszukiwań",
+  "theme.SearchModal.startScreen.recentSearchesTitle": "Ostatnio",
```

```suggestion
  "theme.SearchModal.startScreen.recentSearchesTitle": "Ostatnie wyniki",
```

A bit more clear translations since we have a lot of free space there (it's a header for Algolia recent results..